app/views.py

```diff
@@ -146,6 +157,73 @@ def verify_password(username, password):
     if existing_user.verify_password(password):
         return existing_user
 
+@dataclass
+class Parameters:
+    """ Class for holding feature parameters"""
+    offset: int
+    limit: int
+    crs: str
+    bbox_crs: str
+    bbox: Optional[Tuple[float, float, float, float]] = None
+
+
+def get_validated_parameters(request: Request) -> Parameters:
+    """ Returns the validated query parameters. 
+    IN case of in invalid parameters of parameter values 
+    an exception with 400 status code is raised """
+    for key in request.args.keys():
+        if key not in ["bbox", "offset", "limit", "crs", "bbox-crs"]:
+            logging.error("Unknown parameter %s", key)
+            abort(400)
+    crs = request.args.get("crs", DEFAULT_CRS)
+    bbox_crs = request.args.get("bbox-crs", DEFAULT_CRS)
+    if crs not in GLOBAL_LIST_CRS:
+        logging.error("Unknown crs %s", crs)
+        abort(400)
+    if bbox_crs not in GLOBAL_LIST_CRS:
+        logging.error("Unknown bbox-crs %s", bbox_crs)
+        abort(400)
+    try :
+        limit = int(request.args.get("limit", DEFAULT_LIMIT))
+        if limit < 0:
+            logging.error("Limit must be an positive integer.")
+            abort(400)
+        offset = int(request.args.get("offset", DEFAULT_OFFSET))
+        bbox=request.args.get("bbox", None)
+        if bbox is not None:
+            r = bbox.strip().strip("[]").split(',')
+            if len(r) != 4:
+                logging.error("BBox needs 4 parameters.")
+                abort(400)
+            bbox = tuple(list(map(float, r)))
+        return Parameters(offset=offset, limit=limit, bbox=bbox, crs=crs, bbox_crs =bbox_crs)
+    except ValueError as error:
+        logging.error("Invalid parameter value")
+        logging.error(error)
+        abort(400)
+
+
+def from_WGS84_to_dutchCRS(x: float, y: float)-> Tuple[float, float]:
+    """ Transform a point from WGS84 to the Dutch Coordinate system (28992)"""
+    return  transform(p1=Proj(init='epsg:4326'),
```

Comment:
The wgs84 crs http://www.opengis.net/def/crs/OGC/1.3/CRS84 used by ogc is not exactly the same as epsg:4326, because the axis order is different. OGC uses long-lat, epsg:4326 uses lat-long.